shared/login/forms/intro/index.js

```diff
@@ -52,21 +56,17 @@ const Intro = (props: Props) => (
 const styles = styleSheetCreate({
   banner: {
     backgroundColor: globalColors.blue,
-    padding: isMobile ? globalMargins.small : globalMargins.medium,
+    justifyContent: 'center',
+    minHeight: 40,
+    paddingBottom: globalMargins.tiny,
+    paddingLeft: isMobile ? globalMargins.small : globalMargins.xlarge,
+    paddingRight: isMobile ? globalMargins.small : globalMargins.xlarge,
+    paddingTop: globalMargins.tiny,
+    position: 'absolute',
```

Comment:
@chrisnojima This is because I don't want the content to be pushed down when there's a banner, since we have space for it.